example/controller-registration.yaml

```diff
@@ -18,7 +18,7 @@ spec:
YuP9W9rku9kIcmf+JraggxJcNAi4TvxSTWNnztwEdm9b7uyFc1sHk+Ic6lshh5H8/e81zDNDwBiIwBigQrJAd+e1QcAmEtJlokGN8Uu/JIdlEx5CUmKwzHksTpm45dF1nwj031S94D6LzAFLQ4iw6RTo75NTPtYMyF4SwkMEgcGR4Z0Xi9iAu2awNRxrrgxcF9CeBf79dsDLTod3zEnEFn16gSeND+qBnGUjsNaJigbCGG3e/MyfV9m5LY8vwCWF6f2p7oX1Z2BZesE1foL+wg+X3eAHE/MmfkKf+NHqdJv1pWsZw3jQr5OG0p3oGyPDnXLi+cAyFpv1IIyu7PV0eHk1eHkyOkW843fjq8vRyTDD7ayMfjl+eWJmoFw7cH3gZpPlwAVmRQZSMl5FsIrOwS/l7hh88cAFc6oNu6/y/rP1v4P+zw7wd3UAK/R/r9kqxn8Pm53WXv8/RilT31rFZ/O9tRe4pV6X2TDmodMs4BG7YDFPIif1RVaUepS+L/pvCrxP6qgh63nDehcnM9W8EbvxkCHfe2gX37/GtOI+aco3Mts6zhnSuvKYJ4FQnW7YRyro6CoEm3eTFc5g2fIcL9Nfc+Zcx8nCONXBuF7p8Vxutp/ITCHyN/tS02i/gPk9p2JO6lsdEdefygGrLCegwKSq4BOtIXRjBOIBxFaQtaUIfZNCmBv4qp63qhbQZqVf1uw88f1zDjKYd/D0JpG9LNsejc2nJN89n/O3bGpZyrObyLR0a0HvkPmwV0bACAvDx4Hj+Sx+nnc3dUQltk1oewk5vg+c2KRy2RPDfPeHdiSBq/pR0mDhrY/nDfBeG+UTpKWmYQSiimiw9xDT8Xen14SuItjL5fnv3lcevrI3qWMtZdChaCyV0NoOJMhZCjHIAIq4b+X1hN1HoOCqKL9lkznn1+kEZ/vT8w1ux1podRhl4eUIkyrdSsUKbX1ihRcoCthcL8bLFsbBSW5w+vUy+BiDevmVewGpP6uvw6X7LkP0Vr9ag4UFN6vW5+vh4OXw4mr4eniM93iuTgcnw/H54HhoWKQyLfa7iC/ylu/UY757wab5Wl2PSrefbWV2Ng0P3cBSekcng1fDN0Ds2cXV2ZvhxduL0eUKrX3SkNdRjDSQRmleyKadCCe9xFxPJUTNuukipLofJSGnmbcRF4JXiAR3uN8nl8fnxXBclDeWVMkq+yWBuSXE7yTQe1arWRKJlFzjfrJgJ2jSlAw5O+NPywIbqhmu1p0fOuPrUojKiFmZdaMdHsWim5vzrEtmHufHc9jAcRDxafXujXclAwy3GqLjDgLhDVZekCxy/BI8zWCm/XZ4GkntqauVZ22cICh+SCtknDM2DTag2TlUF+7ypmIKLt3+qtz4ApTp04+ClZdyta10hZ1tkWFvAggecp/P7n9AGuv5+A1GRSTTNYQS1hULqyBtTnqmZVK39ZFWWlw2pYkvTrgLcJ12U7/aSZS3E+Td6a1aGBto3wfk/6pl+/gPKDGwK6JE/grEJHFnbNtAUFX+71GnJP6/z/9/lKI110yQJ+gqlwU1npJWMQU4lB5l46Y1AYMuDRidc/dlJiAvpIB87MjRwwI64Fn8GNAb6vloFUv0cTKpHPAHB3K+BM26/fqPwJt+2A8BVaz/w8NmMf+nddTd5/88SsH0GXNly1mmiZjzyPu3utKPt3nA7FpmB/nAMxZdcJ/tsr53WblR4qNBZ2FWz6uIJ6G07qz1ScO1nPeDTc0gU7xa04BpF4n5AsNEHiupMZs6auz6ixGuKakx4fLRltI6s7kKbeSel6/BtJvoQaKKlZ6AF6uHW9RR8inMnpIQZoitMnNdQH2Vlyq7081q80TUv66vIq/XV9FkBnVsvJP6Xb03FY8F2h8fpQbG1KvScd8WB7kceTlFljzk0HOcwq6VeQXg6h8ayf3Ohdkg9Ay5NF4E6ugWr0pktQWWZJueognM90BLa8ycKJXcHOPknhJyL224PJdLAaW/mvtClfOak2GQU7ZSMYFVCPSq+mWLlVe/8ol6AKtw+dBQScsgMImQPxmiIx25jH7dpzzhTbkhbxJ6y7eac9LM8LaSN52Fbsc0lEJcym2ErEYFksBjh/qb5k3mGEMbGHXaXK3XIvIPUq8vFMs/mZaFLnS8LmXBBgprWXKkof8r6AEz61dYv1KVK+BxLprycazCz72Hfslle/tPq5IHmIBV/l/3qHj/s324v//9OKU0/1sv0r/Yub+QKc9au43Oc/7fduf22RGBJR/7pNM5lGPIx/JlgJNGYMTsdDTwSeZ/5/WvTYld1EDV+m93WsX8n26vuV//j1E2rf90k/6sQZzPzaA/edlh/atLAvKGwW5GQNX673SbxfV/eLRf/49SdPx3hwRwU2OkF0eKPsxY1R+jrJSrjx0useyiLSS5SBuL+iR/bIhBBQZeK/V9fvtGnhkO70IaqJHIM+CQRtCr0Olayh6Yhe0vIpD7wLL9+r8J6QN/B77y/GfF/m91Ovvff3+UUji5xklWvwjrFpZ6fTUsos9/2nW96t/ooMg5dwdZUGT7K6TQ95YWRGaxSyM6Szkqy2XG+pV85qqjdGig4ok6rVICqBp1em4oMOSY2dhetvsSzoCq1r/K6Piw/wCiYv23OodF+7/ZOuru1/9jFJ01XVt3W7nsrrL+0XFs1Pg15ioNSN+fbR2qvJLcNeJSAHnBt9XE5svLr4Wrr8ubq980X3lp5fI2bD4iYE1pLGpGciWiMU99+ilpZsJonxxlzWSq8IZW5vnOhmZrU1/7ZEp9ebsnf/izAZc6/lnbYDVntE9++qVWyACVdSmz1FU3pjmernZ5a75WCMmk9Kp0dIxwGLnmo+kpF+f42/9SpS5/JbRPWJL+/l+mX6AfDAUvf1/d+Fn1Qp65oLM+kUaHqJnn8iAutWUcnrz/owbKF6nSO1aWybdZeePeVMsFWhRH0liLCtt87oW5L/uyL/uyL/uyL/uyL/uyL/uyL/vyEcv/AyD6BOQAeAAA
```

Comment:
Why is the `chart` changed although you didn't change anything under `charts/`?

Reply:
I guess, the CI also fails because of this change:
```
> Check
Executing check-generate
go generate needs to be run:
 M example/controller-registration.yaml
make: *** [Makefile:53: check] Error 1
The command '/bin/sh -c make install-requirements && make VERIFY=true all' returned a non-zero code: 2
```

Reply:
`make generate` produced that result... I guess something in my dependencies / tools.

We have to start using `docker` for those commands as well.